packages/ui/components/proposal/ProposalStatus.tsx

```diff
@@ -1,5 +1,52 @@
-export interface ProposalStatusProps {}
+import clsx from 'clsx'
+import { ComponentType, SVGProps } from 'react'
 
-export const ProposalStatus = ({}: ProposalStatusProps) => {
-  return <p>placeholder</p>
+import { Approved, Open, Rejected } from '@dao-dao/icons'
+
+export enum EProposalStatus {
```

Comment:
Is it a convention to start Enum names with `E`? Separate question: is this convention linted?

Reply:
Not a convention, just wasn't sure what else to call it 😁

Reply:
Let's call it `ProposalStatusEnum`. Took me a sec to figure out what `E` meant.